go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -89,6 +93,10 @@
       end
 
       it "includes an updated go.sum" do
+        ENV["GIT_AUTHOR_NAME"] = previous_git_author_name
+        ENV["GIT_AUTHOR_EMAIL"] = previous_git_author_email
+        ENV["GIT_COMMITTER_NAME"] = previous_git_committer_name
+        ENV["GIT_COMMITTER_EMAIL"] = previous_git_committer_email
```

Comment:
Maybe we can extract methods for this repeated block of code, something like `reset_git_author` and `clear_git_author`?

Reply:
Yeah, this is the one that is causing me the biggest issue.  I was hitting a wall with one test and when I 'fix' that test, I break ten more.  I was considering skipping that test, since it does work locally but not on the CI.

Reply:
See commit https://github.com/dependabot/dependabot-core/pull/10088/commits/39bc17a9f871242b65b62e5b2c4ee081851f9cb3